parallel_cluster/bin/start_cluster.sh

```diff
@@ -6,6 +6,13 @@ echo_stderr(){
     echo "${@}" 1>&2
 }
 
+get_local_ip(){
+  # Returns the local ip address
+  local local_ip
+  local_ip=$(dig +short myip.opendns.com @resolver1.opendns.com)
```

Comment:
Heh, neat trick, I would have used `ip show` or similar (which don't generate an external request) but this works too :)

Reply:
Are you able to replicate `ip show` for getting external IP? I can't seem to find anything on line for it and this seemed to be the shortest solution.

Reply:
Ah, you are right, what I was thinking does not work with NAT or gateways in general, routes nor ARP caches do not expose those either... (conntracking might)... It's fine this way! ;)